src/espIdf/debugAdapter/debugAdapterManager.ts

```diff
@@ -201,9 +236,41 @@ export class DebugAdapterManager extends EventEmitter {
       "esp_debug_adapter",
       "debug_adapter"
     );
+    this.initGdbCommands = [];
+    this.elfFile = "";
+    if (this.currentWorkspace) {
+      this.elfFile = `${path.join(
+        this.currentWorkspace.fsPath,
+        "build",
+        "project-name"
```

Comment:
@brianignacio5 this path is hardcoded, which is causing the error... will add the project name to fix it

Reply:
If you look closely, this for debugAdapter initialization. On extension.ts, when you call debug adapter, you should be getting the new elf file path. Isn't that the case ?

Reply:
Yes this is for initialization, but I was thinking that instead of calling `debugAdapterManager.configureAdapter ` we can have the elf file by default, but anyways we have to call `debugAdapterManager.configureAdapter ` so its fine this way